TACOCAT.py

```diff
@@ -47,15 +46,43 @@
 ## Material Properties
 
 #Thermal Fluid Properties of NaK
-rhoNa = NaK_Prop.rhoNa(Tbulkin + 273.15)
-rhoK = NaK_Prop.rhoK(Tbulkin + 273.15)
-CpNa = NaK_Prop.CpNa(Tbulkin + 273.15)
-CpK = NaK_Prop.CpK(Tbulkin + 273.15)
-Cp = (10**3)*(0.22*CpNa + 0.78*CpK) #J/kg-K
-k = NaK_Prop.k(Tbulkin + 273.15)
-nu = NaK_Prop.nu(Tbulkin + 273.15)
-invrhoNaK = 0.22/rhoNa + 0.78/rhoK
-rho = 1/invrhoNaK #kg/m^3
+CoolantUsed = int(input('Enter the number for the coolant you would like to use: 1. NaK 2. FLiBe 3. FLiNak 4. NaF-ZrF4:  '))
+#Coolants = {'1' : 'NaK_Prop','2' : 'FLiBe_Prop'}

```

Comment:
Do you need this line still?

Reply:
I think I fixed the issue, can you check?